packages/contentstack-export/src/app.js

```diff
@@ -126,9 +127,11 @@ const allExport = async (config, types, branchName) => {
     const stackClient = stack
       .Client(config)
       .stack({ api_key: config.source_stack, management_token: config.management_token });
-    for (const type of types) {
-      const ExportModule = require('./lib/export/' + type);
+
```

Comment:
Please check the extension prettier and its configuration , make sure it is enabled